src/index.njk

```diff
@@ -44,50 +44,91 @@ eleventyNavigation:
   </header>
 
-  <ul class="features__list">
-    <li class="feature feature--clean-design">
-      <img alt="An illustration of a clean window design" src="/images/features/clean-design.svg"/>
-      <div class="feature__info">
-        <h3 class="feature__title">Clean Design</h3>
+  <div class="features__list">
+    <div class="feature-row">
+      <div class="feature-row__image">
+        <img alt="An illustration of a clean window design" src="/images/features/clean-design.svg"/>
       </div>
-    </li>
-    <li class="feature">
-      <img alt="An illustration of a global menu" src="/images/features/global-menu.svg"/>
-      <div class="feature__info">
-        <h3 class="feature__title">Global Menus</h3>
+      <div class="feature-row__content">
+        <h3 class="feature-row__title">Clean Design</h3>
+        <p class="feature-row__text">
+          A distract-free interface that puts your content first. Beautiful transparency, refined typography, and elegant spacing inspired by the best.
+        </p>
       </div>
-    </li>
-    <li class="feature">
-      <img alt="An illustration of Shift and S keys" src="/images/features/shortcuts.svg"/>
-      <div class="feature__info">
-        <h3 class="feature__title">Consistent Shortcuts</h3>
+    </div>
+
+    <div class="feature-row">
+      <div class="feature-row__image">
+        <img alt="An illustration of a global menu" src="/images/features/global-menu.svg"/>
+      </div>
+      <div class="feature-row__content">
+        <h3 class="feature-row__title">Global Menus</h3>
+        <p class="feature-row__text">
+          Save vertical space and access commands consistently. The global menu bar separates application control from window content.
+        </p>
+      </div>
+    </div>
+
+    <div class="feature-row">
+      <div class="feature-row__image">
+        <img alt="An illustration of Shift and S keys" src="/images/features/shortcuts.svg"/>
       </div>
-    </li>
-    <li class="feature">
-      <img alt="An illustration of an app ready to be dragged to Applications folder" src="/images/features/simple-installs.svg"/>
-      <div class="feature__info">
-        <h3 class="feature__title">Simple Installs</h3>
+      <div class="feature-row__content">
+        <h3 class="feature-row__title">Consistent Shortcuts</h3>
+        <p class="feature-row__text">
+          Muscle memory matters. Use the standard Command-key shortcuts you already know and love across the entire system.
+        </p>
       </div>
-    </li>
-    <li class="feature">
-      <img alt="Four familiar folders - Application, System, Library, and Users" src="/images/features/familiar-folders.svg"/>
-      <div class="feature__info">
-        <h3 class="feature__title">Familiar Folders</h3>
+    </div>
+
+    <div class="feature-row">
+      <div class="feature-row__image">
+        <img alt="An illustration of an app ready to be dragged to Applications folder" src="/images/features/simple-installs.svg"/>
```

Comment:
On the dark bg, the black letters and arrows are very hard to see. Can we make them white in dark mode?